pgcli/pgexecute.py

```diff
@@ -6,10 +6,11 @@
 import psycopg2.extensions as ext
 import sqlparse
 import pgspecial as special
+from collections import namedtuple
 from .packages.function_metadata import FunctionMetadata
 from .encodingutils import unicode2utf8, PY2
 
-
+ForeignKey = namedtuple('ForeignKey', ['parentschema', 'childschema', 'parenttable', 'childtable', 'parentcolumns', 'childcolumns'])
```

Comment:
They're slightly different in that FKs from the DB can contains multiple columns, whereas the FKs stored in ColumnMetadata are a relationship between only two columns (for simplicity).